### ISSUE TITLE
crypto-common 0.1.4 does not work with MSRV (does not work with rust 1.48)

### ISSUE THREAD

**Issue by catenacyber** [RustCrypto/traits#1047]
Version 0.1.3 worked fine as in 
https://github.com/OISF/suricata/runs/7155746336?check_suite_focus=true

Version 0.1.4 fails as in 
https://github.com/OISF/suricata/runs/7177440733?check_suite_focus=true

```
Downloaded crypto-common v0.1.4
error: failed to parse manifest at `/github/home/.cargo/registry/src/github.com-1ecc6299db9ec823/crypto-common-0.1.4/Cargo.toml`
Caused by:
  feature `resolver` is required
  this Cargo does not support nightly features, but if you
  switch to nightly channel you can add
  `cargo-features = ["resolver"]` to enable this feature
```

**Comment by gruve-p** [comment]
Same problem for me (rust 1.47)

Version 0.1.3 worked fine as in
https://github.com/Groestlcoin/rust-groestlcoin/actions/runs/2602126182

Version 0.1.4 fails as in
https://github.com/Groestlcoin/rust-groestlcoin/actions/runs/2640925514

```
Downloaded crypto-common v0.1.4
error: failed to parse manifest at `/home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/crypto-common-0.1.4/Cargo.toml`
Caused by:
  feature `resolver` is required
  this Cargo does not support nightly features, but if you
  switch to nightly channel you can add
  `cargo-features = ["resolver"]` to enable this feature
Error: Process completed with exit code 101.
```

**Comment by tarcieri** [comment]
Fixed by #1049

**Comment by catenacyber** [comment]
Thanks @tarcieri it looks like 0.1.5 is working fine :-)
